OHHTTPStubs/OHHTTPStubs.xcodeproj/project.pbxproj

```diff
@@ -1685,18 +1686,22 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				INFOPLIST_FILE = "UnitTests/UnitTests-Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MACOSX_DEPLOYMENT_TARGET = 10.9;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				PRODUCT_BUNDLE_IDENTIFIER = "alisoftware.OHHTTPStubs-tvOS-Fmk-Tests";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SDKROOT = appletvos;
-				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```

Comment:
Was this setting removed when supporting the Xcode recommended settings?

Reply:
all of this changes was done automatically by "update xcode project to recommended settings"

"project level" is more preferred instead of framework/target settings, because project can be changed by every developer instead of only code owner
example, i don't want to see any warnings in my project and than i can setup any MY project settings which will inherited by YOURs target. proposal: ios/macos version must be removed and xcode will use "default" version based on the parent project

and as proposal you must review your demo/project/target/framework settings, because as you see many of them are old legacy from the previous versions